### PR TITLE
update documentation with fastapi

### PR DESCRIPTION
**Pull request by mgaonach**

### CODE REVIEW

File: docs/run.md [hunk 1]
```diff
@@ -43,30 +41,16 @@ cd ..
...
-Now you can go to [http://localhost:5000/ispyb/api/v1/docs](http://localhost:5000/ispyb/api/v1/docs) and explore py-ispyb via swagger ui. Please see the [routes section](routes.md) for more information.
+Now you can go to [http://localhost:8000/docs](http://localhost:8000/docs) and explore py-ispyb via OpenAPI ui. Please see the [routes section](routes.md) for more information.
 
-For requests use the token in the `Authorization` header: `Bearer YOUR_JWT_TOKEN`.
+For requests use the token in the `Authorization` header: `YOUR_JWT_TOKEN`.
 Please see the [authentication and authorization section](auth.md) for more information.
 For example to retrieve proposals call:
 
 ```bash
 curl -X GET -H 'Authorization: Bearer YOUR_JWT_TOKEN' -i http://localhost:5000/ispyb/api/v1/proposals
```
Comment (stufisher) [inline docs/run.md:55]: This ones :8000 rather than :5000 too

File: docs/auth.md [hunk 2]
```diff
@@ -89,34 +88,30 @@ curl -X GET -H 'Authorization: Bearer YOUR_TOKEN' -i http://localhost:5000/ispyb
 
 ---
 
-## Decorators
+## Authorization dependancies
```
Comment (stufisher) [inline docs/auth.md:91]: dependencies

**Review by stufisher — COMMENTED** [review]
Looks good to me

File: docs/run.md (hunk 1 above)
Comment (mgaonach) [reply docs/run.md:55]: Thanks!

**Pull request merged**